doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+  .. _ref_guide_wsl:
+
+
+ANSYS on Windows Subsystem for Linux and Docker
+###############################################
+
+This guide shows you how to use Ansys products, more specifically PyMAPDL, in the Windows Subsystem for Linux (WSL).
+WSL is a compatibility layer for running Linux binary executables natively on Windows 10, Windows 11, and Windows Server 2019 
+(`Wikipedia-WSL <https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux>`_).
+
+This guide walks you through the installation of WSL in Windows, and shows how to use it together with ANSYS, PyMAPDL and docker.
+
+For more information about WSL please visit `What is the Windows Subsystem for Linux? <https://docs.microsoft.com/en-us/windows/wsl/about>`_. 
+
+.. warning:: **Disclaimer:** This guide is still in Alpha state. Please proceed with caution. 
+
+
+.. warning:: This guide hasn't been tested with a VPN connection.
+
+
+Running ANSYS on WSL 
+*********************
+
+Install WSL2
+=============
+
+Follow the instructions in the official Microsoft website `Install WSL <https://docs.microsoft.com/en-us/windows/wsl/install/>`_ .
+
+It is recommended you use WSL2 over WSL1. 
+
+Install CentOS7 WSL Distribution
+=================================
+
+This is the recommended distribution to work with PyANSYS.
+
+You can install it using an unofficial WSL distribution from `<https://github.com/wsldl-pg/CentWSL/>`_ .
+
+.. note:: 
+
+    It seems the above project has been discontinued.
+    A more updated CentOS distribution can be found in `<https://github.com/mishamosher/CentOS-WSL/>`_ , however it has not been tested yet.
```

Comment:
```suggestion
    It seems that the above project has been discontinued. While a more updated CentOS distribution is hosted at `<https://github.com/mishamosher/CentOS-WSL/>`_ , it has not been tested yet.
```

Reply:
I'm assuming that this will all be rewritten. It's not normal to say use this--oh no, you can't, but you can use this, which might not work.

Reply:
Fair enough. I will rewrite this.

Reply:
Re-written.